src/main/java/com/cognifide/SpringBootApplication.java

```diff
@@ -0,0 +1,36 @@
+package com.cognifide;
+
+import com.cognifide.domain.Book;
+import com.cognifide.repository.BookRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.context.annotation.Bean;
+
+/**
+ * Created by cognifide.nowak on 01.04.2016.
+ */
+@org.springframework.boot.autoconfigure.SpringBootApplication
+public class SpringBootApplication {
+
+    @Autowired
+    private BookRepository bookRepository;
+
+    public static void main(String[] args) throws Exception {
+        SpringApplication.run(SpringBootApplication.class, args);
+    }
+
+    @Bean
+    public CommandLineRunner dataInitializer(BookRepository br) {
+        return new CommandLineRunner() {
+            public void run (String... args) throws Exception {
+                // initialize bookRepository
```

Comment:
Comments can get outdated pretty fast. However, this one is valueable. Maybe you can place this knowledge (that tells pupropse of instructions below) directly to code?